### PR TITLE
Update nouns.json

### PR DESCRIPTION
**Pull request by encosllo**
Catalan forms.

### CODE REVIEW

File: nouns.json [hunk 1]
```diff
@@ -40,6 +44,7 @@
         "RU": { "atom": "нётерова"    , "pstn": "before" },
       },
       "433b071e": {
+        "CA": { "atom": "localmentnoetherià"    , "pstn": "after"  },
```
Comment (thosgood) [inline nouns.json]: should there be a space in between "localment" and "noetherià"?

File: nouns.json [hunk 2]
```diff
@@ -113,7 +125,7 @@
     },
   },
   "0677022e": {
-    "root": {
+    "root":
```
Comment (thosgood) [inline nouns.json]: I think you accidentally deleted a `{` here! It should be `"root": {`

**Review by thosgood — CHANGES_REQUESTED** [review]

File: nouns.json (hunk 1 above)
Comment (encosllo) [reply nouns.json]: Yes, it must be "localment noetherià". Sorry!

File: nouns.json (hunk 2 above)
Comment (encosllo) [reply nouns.json]: You are right.

**Review by thosgood — APPROVED** [review]

**Pull request merged**